recipes/minizip-ng/all/conanfile.py

```diff
@@ -162,7 +162,7 @@ def package_info(self):
         self.cpp_info.set_property("pkg_config_name", "minizip")
 
         # TODO: back to global scope in conan v2 once cmake_find_package_* generators removed
-        prefix = "lib" if is_msvc(self) or self._is_clang_cl else ""
+        prefix = "lib" if Version(self.version) < "4.0.7" and (is_msvc(self) or self._is_clang_cl) else ""
```

Comment:
For anyone else curious, the corresponding minizip-ng change is [here](https://github.com/zlib-ng/minizip-ng/commit/b887f8104bceaf0d6cd0bb3f24e6f17eb68cb73d).

Reply:
Thanks for your help 😄